code/game/jobs/job/shipside.dm

```diff
@@ -65,30 +65,32 @@ Come hell or high water, you are going to be there for them."}
 
 
 //Executive Officer
-/datum/job/command/executive
-	title = "Executive Officer"
-	comm_title = "XO"
-	paygrade = "O3"
-	flag = ROLE_EXECUTIVE_OFFICER
+/datum/job/command/fieldofficer
+	title = "Field Officer"
+	comm_title = "FO"
+	paygrade = "MAJ"
+	flag = ROLE_FIELD_OFFICER
 	flags_startup_parameters = ROLE_ADD_TO_DEFAULT|ROLE_ADD_TO_MODE|ROLE_ADMIN_NOTIFY
-	skills_type = /datum/skills/XO
+	skills_type = /datum/skills/FO
 	access = ALL_MARINE_ACCESS
 	minimal_access = ALL_MARINE_ACCESS
 	equipment = TRUE
 
-/datum/job/command/executive/generate_equipment(mob/living/carbon/human/H)
+/datum/job/command/fieldofficer/generate_equipment(mob/living/carbon/human/H)
 	H.equip_to_slot_or_del(new /obj/item/device/radio/headset/almayer/mcom(H), SLOT_EARS)
 	H.equip_to_slot_or_del(new /obj/item/clothing/under/marine/officer/exec(H), SLOT_W_UNIFORM)
 	H.equip_to_slot_or_del(new /obj/item/clothing/head/cmcap(H), SLOT_HEAD)
 	H.equip_to_slot_or_del(new /obj/item/clothing/shoes/marine(H), SLOT_SHOES)
 	H.equip_to_slot_or_del(new /obj/item/storage/backpack/marine/satchel(H), SLOT_BACK)
-	H.equip_to_slot_or_del(new /obj/item/storage/belt/gun/m4a3/vp70(H), SLOT_BELT)
+	H.equip_to_slot_or_del(new /obj/item/storage/belt/gun/m4a3/vp78(H), SLOT_BELT)
```

Comment:
This pistol currently is VERY OP. Like, at least three times better than the mateba.